terminology_bot/bot.py

```diff
@@ -0,0 +1,404 @@
+            update.message.reply_text(text, reply_markup=ReplyKeyboardRemove())
+            return self.DESCRIPTION
+
+        elif option == 'Synonyms':
+            text = _('List synonyms of the term "{}" separating them with comma.'.format(self.cur_term.name))
+            update.message.reply_text(text, reply_markup=ReplyKeyboardRemove())
+            return self.SYNONYMS
+
+        elif option == 'Similar words':
+            text = _('List words similar with the term "{}" separating them with comma.'.format(self.cur_term.name))
+            update.message.reply_text(text, reply_markup=ReplyKeyboardRemove())
+            return self.SIMILARS
+
+        elif option == 'Image':
+            text = _('Let\'s upload an image for the term "{}".'.format(self.cur_term.name))
+            update.message.reply_text(text, reply_markup=ReplyKeyboardRemove())
+            return self.IMAGE
+
+        elif option == 'Audio':
+            text = _('Let\'s upload an audiofile for the term "{}".'.format(self.cur_term.name))
+            update.message.reply_text(text, reply_markup=ReplyKeyboardRemove())
+            return self.AUDIO
+
+        elif option == 'Video':
+            text = _('Let\'s upload a video for the term "{}".'.format(self.cur_term.name))
+            update.message.reply_text(text, reply_markup=ReplyKeyboardRemove())
+            return self.VIDEO
+
+        else:
+            text = _('Feel free to choose.')
+            update.message.reply_text(text, reply_markup=self.reply_menu_keyboard)
+            return self.CHOOSE_OPTION
+
+    def pos_tag(self, bot, update):
+        """
+        Saves pos-tag of the current term to DB
+        """
+        user = update.message.from_user
+        pos_tag = update.message.text
+
+        logger.info('User %s chose pos-tag "%s"', user.first_name, pos_tag)
+
+        self.term_collection.update(self.cur_term.id, {'pos_tag': pos_tag})
+
+        update.message.reply_text(_('I see!'), reply_markup=self.reply_menu_keyboard)
+        return self.CHOOSE_OPTION
+
+    def description(self, bot, update):
+        """
+        Saves description of the current term to DB
+        """
+        user = update.message.from_user
+        dscr = update.message.text
+
+        logger.info('User %s gave a description to the term "%s"', user.first_name, self.cur_term.name)
+
+        self.term_collection.update(self.cur_term.id, {'description': dscr})
+
+        update.message.reply_text(_('Good work!'), reply_markup=self.reply_menu_keyboard)
+        return self.CHOOSE_OPTION
+
+    def image(self, bot, update):
+        """
+        Saves image of the current term to DB
+        """
+        user = update.message.from_user
+        photo_file = bot.get_file(update.message.photo[-1].file_id)
+
+        filename_sha1 = hashlib.sha1(bytes(f'image_{self.cur_term.name}', encoding='utf8')).hexdigest()
+        photo_file.download(f".{params['multimedia_dir']}/{filename_sha1}")
```

Comment:
f".{params['multimedia_dir']}/{filename_sha1}" ==> f"{params['multimedia_dir']}/images/{filename_sha1}.jpg"

Reply:
Fixed in commit https://github.com/annkupriyanova/wiki-framework/pull/5/commits/fde248160994c48824d736f63a9bfee3b68896bf